### PR TITLE
Paging params in API docs

### PR DESCRIPTION
**Pull request by markus101**
#### Database Migration
NO

#### Description
Adds paging params to API docs.

#### Todos
- [ ] Tests
- [ ] Wiki Updates

#### Issues Fixed or Closed by this PR

* #6003

### CODE REVIEW

File: src/NzbDrone.Host/Startup.cs [hunk 1]
```diff
@@ -158,6 +158,8 @@ public void ConfigureServices(IServiceCollection services)
                 {
                     { apikeyQuery, Array.Empty<string>() }
                 });
+
+                c.DescribeAllParametersInCamelCase();
```
Comment (markus101) [inline src/NzbDrone.Host/Startup.cs:162]: This gives camelCase names for properties on `PagingRequestResource` and other models

File: src/Sonarr.Api.V3/Blocklist/BlocklistController.cs [hunk 2]
```diff
@@ -23,9 +23,9 @@ public class BlocklistController : Controller
 
         [HttpGet]
         [Produces("application/json")]
-        public PagingResource<BlocklistResource> GetBlocklist()
+        public PagingResource<BlocklistResource> GetBlocklist([FromQuery] PagingRequestResource paging)
         {
-            var pagingResource = Request.ReadPagingResourceFromRequest<BlocklistResource>();
+            var pagingResource = new PagingResource<BlocklistResource>(paging);
```
Comment (markus101) [inline src/Sonarr.Api.V3/Blocklist/BlocklistController.cs:28]: This drops support for `FilterKey` and `FilterValue` as of right now params for filtering, but we could probably add that behaviour back. This was originally added for backwards compatibility with v2 API.

File: src/Sonarr.Api.V3/openapi.json [hunk 3]
```diff
@@ -381,6 +381,50 @@
...
+          },
+          {
+            "name": "sortKey",
+            "in": "query",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "sortDirection",
```
Comment (markus101) [inline src/Sonarr.Api.V3/openapi.json:411]: Getting a default for the sort direction requires inlining enum values, which doesn't seem great.

File: src/Sonarr.Api.V3/openapi.json [hunk 4]
```diff
@@ -381,6 +381,50 @@
...
+            "name": "pageSize",
+            "in": "query",
+            "schema": {
+              "type": "integer",
+              "format": "int32",
+              "default": 10
+            }
+          },
+          {
+            "name": "sortKey",
```
Comment (markus101) [inline src/Sonarr.Api.V3/openapi.json:404]: This changes depending on the resource so no default value is set.

File: src/Sonarr.Api.V3/Blocklist/BlocklistController.cs (hunk 2 above)
Comment (Qstick) [reply src/Sonarr.Api.V3/Blocklist/BlocklistController.cs:28]: I think this is likely fine with the v4 bump

**Review by Qstick — APPROVED** [review]

**Pull request merged**